editor/src/components/navigator/navigator-item/item-preview.tsx

```diff
@@ -18,12 +19,21 @@ export const ItemPreview: React.FunctionComponent<ItemPreviewProps> = betterReac
     // 3 - if it's a component or not
     // 4 - if it's a placeholder or not
     // 5 if it's generated or not
```

Comment:
`// 6 - if we can generate a preview for it`?
I don't actually understand what the 6th 3rd thing is here - if we return `null` do we just not have a preview?

Reply:
I removed these comments because this is not how the icons work now